src/config.cc

```diff
@@ -0,0 +1,23 @@
+#include <fstream>
+#include <iostream>
+#include <filesystem>
+#include "config.hh"
+
+int main (const int argc, const char* argv[]) {
+  auto base = std::filesystem::path(argv[1]);
+  std::filesystem::path dest = { base / ".ssc.dat"};
+
+  std::ofstream ws(dest, std::ios::out | std::ios::binary);
+  if (!ws) return 1;
+
+  using namespace SSC;
+
+  #include "ssc.conf" // NOLINT
+
+  ws.write((char*) &config, sizeof(Config));
```

Comment:
we should enclose `config` in a function:

```c++
static void configure (Config& config) {
  #include "ssc.conf" // NOLINT
}
```

The declaration of `config` should be in `main()` or static to the file so the caller need not worry about it. They'll still be able to do:

```c++
config = {
  // fields
};
```